test/README.md

```diff
@@ -126,3 +130,243 @@ and `setInterval`).
 | Runs on CI |
 |:----------:|
 | No         |
+
+
+## Common.js
```

Comment:
I changed this to read 'Common module API' to match the wording used in the table of contents. Is that cool?